### PR TITLE
Remove images from Intervention Type Groups

### PR DESCRIPTION
**Pull request by iant**

### CODE REVIEW

**Comment by iant** [comment]
@ldodds @urbanwide I'm not sure on this one - it doesn't look like Intervention Type Group images are used anywhere except on the form? (I may be wrong) Should we remove instead?

File: app/views/admin/intervention_type_groups/show.html.erb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <h1>Intervention Category</h1>
-<p><strong>Title:</strong> <%= @intervention_type_group.title %></p>
+<p><strong>Title:</strong> <%= @intervention_type_group.name %></p>
```
Comment (urbanwide) [inline app/views/admin/intervention_type_groups/show.html.erb:2]: Wondering why this changed?

Comment (iant) [reply app/views/admin/intervention_type_groups/show.html.erb:2]: sorry @urbanwide  I should have left a comment explaining that.  it's a bug fix - no `title` on InterventionTypeGroups 
https://rollbar.com/energysparks/EnergySparksProduction/items/1642/
(prob should have gone in a different pr but i only noticed the issue when testing this one)

Comment (urbanwide) [reply app/views/admin/intervention_type_groups/show.html.erb:2]: Ahh cool, makes sense :-) Thanks for explaining

**Pull request merged**